MovieGallery/wwwroot/css/site.css

```diff
@@ -71,4 +71,9 @@ body {
     width: 100%;
     white-space: nowrap;
     line-height: 60px; /* Vertically center the text there */
+}
+.table {
```

Comment:
Added css for both Movies and shows table to have fixed width as by default the table width was auto and looked unaligned when width of the corresponding column in Movies and Shows did not match.

https://www.w3schools.com/cssref/pr_tab_table-layout.asp

Reply:
Good to put them here, in the external CSS page.